src/utilities/buildASTSchema.js

```diff
@@ -280,8 +255,8 @@ export function buildASTSchema(ast: DocumentNode): GraphQLSchema {
     });
   }
 
-  function getObjectType(typeNode: TypeDefinitionNode): GraphQLObjectType {
-    const type = typeDefNamed(typeNode.name.value);
+  function getObjectType(name: string): GraphQLObjectType {
```

Comment:
Hmm, this is an API change. Have we done a sweep to see if we're breaking anything? is `getObjectType` only used in this file?

Reply:
@asiandrummer Yes, since it doesn't have `export`